libs/form-component/src/app-components/Table/Table.module.css

```
@@ -78,3 +78,16 @@
   color: var(--semantic-text-neutral-subtle);
 }
+
+.visuallyHidden {
+  border: none;
+  padding: 0;
+  margin: 0;
+  position: absolute;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  clip: rect(1px 1px 1px 1px);
```

Stylelint (17.11.1)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)